audformat/core/database.py

```diff
@@ -12,6 +12,7 @@
 from audformat.core.index import segmented_index
 from audformat.core.column import Column
 from audformat.core.common import HeaderBase, HeaderDict
+from audformat.core.define import LICENSE_URLS
```

Comment:
We already do `from audformat.core import define`, so maybe we can remove and replace ` LICENSE_URLS` with `define.LICENSE_URLS` below?

Reply:
You are right, I changed it accordingly.